### PR TITLE
Grafana Plugins & Cluster Dashboard

### LINKED ISSUE #1782: [Grafana/Prom Customizing] Allow the use of grafana plugins
It would be good to have the possibility to install grafana plugins, which could be maybe done with such an init container for example:

```
      {{- if .Values.grafana.plugins }}
      initContainers:
      - name: install-plugins
        image: {{ .Values.grafana.image.repository }}:{{ .Values.grafana.image.tag }}
        env:
        - name: GF_PLUGIN_DIR
          value: /data/plugins
        command:
          - /bin/sh
          - -c
          - |
            mkdir -p /data/plugins
            {{- range .Values.grafana.plugins }}
            ./bin/grafana-cli plugins install {{ . }}
            {{- end }}
        volumeMounts:
        - mountPath: /data
          name: grafana-storage
          readOnly: false
      {{- end }}
```

Issue extracted from https://github.com/kubermatic/kubermatic/pull/1775#issuecomment-413459153

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This PR introduces a new public Docker image for carrying Grafana plugins, `quay.io/kubermatic/grafana-plugins`. We use some of the new plugins in the new, fancy User Cluster dashboard already:

screenshot-2019-10-11-142255

This PR also moves the NGINX dashboard to a better place and makes the dashboard verification job stricter by not just requiring properly formatted JSON, but also that all the other normalizations have been applied.

Fixes #1782 (mostly; not all plugins can be installed this way, but this is an 80/20 solution to the problem and customers can easily create their own Docker image with plugins).

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: containers/grafana-plugins/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.10
+
+RUN mkdir -p /plugins
+WORKDIR /plugins
+
+RUN wget -O- https://grafana.com/api/plugins/grafana-piechart-panel/versions/1.3.9/download | unzip - && \
```
Comment (kron4eg) [inline containers/grafana-plugins/Dockerfile:6]: I'd parametrize version, with `ARG` instruction.

Comment (xrstf) [reply containers/grafana-plugins/Dockerfile:6]: I hate Docker args when building images. :-D I also don't see an advantage. Either we manage simple, working URLs inside the Dockerfile or split the stuff into two or more places.

**Review by kron4eg — COMMENTED** [review]
/lgtm

**Pull request merged**